### PR TITLE
Add smartmontools to global program list, alice config changes

### PR DESCRIPTION
**Pull request by ahuston-0**
Adds smartmontools to the global program list so we can do SMART tests, also make some changes to my personal config.

### CODE REVIEW

File: users/alice/default.nix [hunk 1]
```diff
@@ -2,6 +2,13 @@
 import ../default.nix {
   inherit pkgs lib config name;
   publicKeys = [
+    #dennis
```
Comment (DerDennisOP) [inline users/alice/default.nix:5]: thats your key bruh

File: users/alice/home.nix [hunk 2]
```diff
@@ -33,7 +51,15 @@
...
-  programs.zsh.enable = true;
+  programs = {
+    zsh.enable = true;
+
+  starship.enable = true;
+  fzf = {
+    enable = true;
+    enableZshIntegration = true;
+  };
+  };
```
Comment (DerDennisOP) [inline users/alice/home.nix:62]: fix formatting

**Review by DerDennisOP — COMMENTED** [review]
almost good

File: users/alice/default.nix (hunk 1 above)
Comment (ahuston-0) [reply users/alice/default.nix:5]: no its the one i created for your servers lol

Comment (ahuston-0) [reply users/alice/default.nix]: changed it to say photon instead

File: users/alice/home.nix (hunk 2 above)
Comment (ahuston-0) [reply users/alice/home.nix]: ran `nixpkgs-fmt` manually

**Review by DerDennisOP — APPROVED** [review]
looks good

**Pull request merged**